### PR TITLE
Update grapesjs to more recent version (Mautic 5)

### PR DESCRIPTION
**Pull request by LordRembo**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

_**[The equivalent for M4 is here](https://github.com/mautic/mautic/pull/13285)**_

This updates GrapesJs to the latest version (at time of writing) as well as its dependencies.
It also updates Parcel (the build tool) to v.2 (v1 is no longer supported)
Because it relies on an update of `grapesjs-preset-mautic`, and that hasn't been merged yet ([see this PR](https://github.com/mautic/grapesjs-preset-mautic/pull/49)), I loaded [my forked branch](https://github.com/mautic/mautic/pull/13283/files#diff-7bfbc6d869d142cddd9b934e5de8527cbe7f454f64c4e393053f26436ee799c5R28). But that shouldn't stop anyone from testing.

Some changes of note:
* I added a `rebuild` and `rebuild-dev` npm commands because a normal build doesn't actually override the previous files in the `dist` folder because Parcel has cache files. This essentially does a force clear before building. I found these handy to have while developing.
* The sidebar icons in the Editor are now SVG instead of an icon font, they look slightly different and scale differently. I've set a max size to come close to the previous styling.
* The colors are a bit darker/brighter but more uniform. The theming of the newsletter and page presets works a bit differently now (no loose css files). Overall, the styling is now more uniform across the various Editors: mjml newsletter, html newsletter and Page. I have overwritten the hardcoded styles a bit to get the green hover color back, instead of red. 
* The Page Editor sidebar layout changed as some blocks got rearranged and regrouped, so I moved some things around to look more like the previous layout. Works better for existing users, I think.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. For a new Mautic install (or when using Gitpod), https://github.com/mautic/mautic/issues/13510 (recent merges of 5.x seems to have broken this)
3. Make or edit a new Email: check one written in MJML and one written in html (eg. Paprika and Blank themes)
4. Make or edit a new Landing Page
5. You should still be able to drag&drop Sections and Blocks, the Styles should still work as expected and you can still use the ckeditor to change text
6. Styles don't get lost when opening/closing the Editor
7. Mails/Pages can still be saved and keep their changes when refreshing the page and reopening

### CODE REVIEW

**Review by kuzmany — CHANGES_REQUESTED** [review]
I am on dev enviroment. I call composer install to be sure all was procesed. I see two issues at the moment:

- Save spinner icon is still active even If background task was done properly

image

Dynamic conten has some stylesheets issue for labels 

image

**Comment by LordRembo** [comment]
@kuzmany Are you editing a mail or a Page? What theme are you using?

**Comment by LordRembo** [comment]
@kuzmany I think I fixed the styling issues, overwriting the builder theme colors for nested panels like that.
I'm not sure about the spinner though. I've tried saving mails and pages and those behave as expected. What are the steps you did to get a spinner that keeps loading?

**Comment by LordRembo** [comment]
@kuzmany I think issues with the 'save' spinner are not related to this PR or GrapesJs updates themselves but logic in [`grapesjs-mautic-preset`](https://github.com/mautic/grapesjs-preset-mautic). The 'Apply' logic is part of that dependency and it's not something I touched. There was another change merged today, related to that button though, so maybe it's related.
Can you check you have the same issue in Mautic 5.x ?

**Review by kuzmany — DISMISSED** [review]
Everything works now as expected. See nothing wrong 👍

**Comment by pjrobertson** [comment]
This is great. I was just looking at making a PR to upgrade grapejs, but you've done it. Thanks @LordRembo ! I hope this PR can be merged soon.

**Comment by LordRembo** [comment]
@pjrobertson We're getting there! Hopefully this'll be done by next week. Once the dependency is merged, this PR should get in as well (seeing as it's successfully tested)

File: plugins/GrapesJsBuilderBundle/package.json [hunk 1]
```diff
@@ -26,20 +40,34 @@
...
+    "grapesjs-navbar": "^1.0.2",
     "grapesjs-parser-postcss": "^0.1.1",
+    "grapesjs-plugin-ckeditor": "^0.0.10",
     "grapesjs-plugin-ckeditor5": "^0.0.33",
-    "grapesjs-preset-mautic": "github:mautic/grapesjs-preset-mautic#ckeditor5",
-    "grapesjs-preset-newsletter": "^0.2.21",
-    "grapesjs-preset-webpage": "^0.1.11"
+    "grapesjs-plugin-export": "^1.0.11",
+    "grapesjs-plugin-forms": "^2.0.5",
+    "grapesjs-preset-mautic": "github:LordRembo/grapesjs-preset-mautic#update-grapesjs-compatibility-m5",
```
Comment (adiux) [inline plugins/GrapesJsBuilderBundle/package.json]: Is this intentional? Looks like you link it to your fork.

File: plugins/GrapesJsBuilderBundle/package.json [hunk 2]
```diff
@@ -26,20 +40,34 @@
   },
   "homepage": "https://github.com/mautic/plugin-grapesjs-builder#readme",
   "dependencies": {
-    "grapesjs": "^0.17.29",
-    "grapesjs-mjml": "^0.5.8",
+    "grapesjs": "^0.21.8",
```
Comment (adiux) [inline plugins/GrapesJsBuilderBundle/package.json]: there are some breaking changes that we are affected by. Are you aware of that? https://github.com/GrapesJS/grapesjs/releases/tag/v0.20.1

File: plugins/GrapesJsBuilderBundle/package.json (hunk 1 above)
Comment (LordRembo) [reply plugins/GrapesJsBuilderBundle/package.json]: @adiux Yes, as I explain in the intro of this PR, there is a separate PR for this dependency that needs to be merged so I can remove my fork. That's why this PR is ~~kept as Draft~~ marked as Blocked.

File: plugins/GrapesJsBuilderBundle/package.json (hunk 2 above)
Comment (LordRembo) [reply plugins/GrapesJsBuilderBundle/package.json]: @adiux Yes, I know. Those changes have been addressed by changes in the other files (eg. builder service but specifically the update to the grapes js preset mautic dependency)

**Comment by code5rick** [comment]
Tested, and it seems OK. It is only working for the landing pages as you said @LordRembo

**Comment by LordRembo** [comment]
~~There's some JS issues coming from merging the 5.x branch (eg. I'm seeing ckeditor errors), which seems to have broken the whole Builder implementation. So until #13510 is fixed (and possibly other JS errors) this one's kinda hard to test.
Still, would be useful if someone did run this and check what kind of errors they see.~~

**Comment by LordRembo** [comment]
This should all be working now. I've just updated the description to point to a bug coming in from 5.x where GrapesJS isn't enabled by default.

**Comment by code5rick** [comment]
@LordRembo I tested your PR, and it works fine. However, it just needs some adjustments. It appears to only work for Landing pages, which I can confirm.

When we want to add a new module, in the blocks section, it seems that the icon is not displayed correctly and shows a "b" for the Button module.
image

Another issue I noticed is with the maps module. In the editor, it appears to have a 100% width, but in reality, it has a specific width that isn't reflected in the builder.

Frame 49

Additionally, I observed that the new version seems faster and saves quickly. Hopefully, it's not just a placebo effect.

You've got this! 🚀

**Comment by LordRembo** [comment]
@code5rick Thanks for the feedback! Very useful info here.
The updated builder should work for Emails as well as the Landing pages (although they both have some differences in Blocks by default). I'll take another look to see what's happening with the default blocks compared with added ones (like the Button block), as well as the maps rendering. Probably gonna make some time on Friday.

**Comment by code5rick** [comment]
@LordRembo you are correct, my apologies! I didn't try using the editor with the email before.

On the email side, there are some bugs that I've encountered:

- When we select any email theme and save it without making any alterations, upon previewing, the template doesn't display properly, showing only white space. However, if we make any alteration to the template, such as adding a letter or module, it displays normally in preview mode.

- When using the theme "Brienz," a standard Mautic theme, there seems to be a bug when saving.
image
System log:
`
#0 [internal function]: Mautic\\CoreBundle\\ErrorHandler\\ErrorHandler->handleError(8, 'PHP Notice - ic...', '/var/www/html/a...', 420)
#1 /var/www/html/app/bundles/CoreBundle/Helper/InputHelper.php(420): iconv('UTF-8', 'Windows-1252', '<!DOCTYPE html>...')
#2 /var/www/html/app/bundles/CoreBundle/Helper/InputHelper.php(142): Mautic\\CoreBundle\\Helper\\InputHelper::html('<!DOCTYPE html>...', false)
#3 /var/www/html/app/bundles/CoreBundle/Form/EventListener/CleanFormSubscriber.php(35): Mautic\\CoreBundle\\Helper\\InputHelper::_(Array, Array)
#4 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(230): Mautic\\CoreBundle\\Form\\EventListener\\CleanFormSubscriber->preSubmitData(Object(Symfony\\Component\\Form\\Event\\PreSubmitEvent), 'form.pre_submit', Object(Symfony\\Component\\EventDispatcher\\EventDispatcher))
#5 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(59): Symfony\\Component\\EventDispatcher\\EventDispatcher->callListeners(Array, 'form.pre_submit', Object(Symfony\\Component\\Form\\Event\\PreSubmitEvent))
#6 /var/www/html/vendor/symfony/event-dispatcher/ImmutableEventDispatcher.php(33): Symfony\\Component\\EventDispatcher\\EventDispatcher->dispatch(Object(Symfony\\Component\\Form\\Event\\PreSubmitEvent), 'form.pre_submit')
#7 /var/www/html/vendor/symfony/form/Form.php(568): Symfony\\Component\\EventDispatcher\\ImmutableEventDispatcher->dispatch(Object(Symfony\\Component\\Form\\Event\\PreSubmitEvent), 'form.pre_submit')
#8 /var/www/html/vendor/symfony/form/Extension/HttpFoundation/HttpFoundationRequestHandler.php(110): Symfony\\Component\\Form\\Form->submit(Array, true)
#9 /var/www/html/vendor/symfony/form/Form.php(503): Symfony\\Component\\Form\\Extension\\HttpFoundation\\HttpFoundationRequestHandler->handleRequest(Object(Symfony\\Component\\Form\\Form), Object(Symfony\\Component\\HttpFoundation\\Request))
#10 /var/www/html/app/bundles/CoreBundle/Controller/AbstractFormController.php(164): Symfony\\Component\\Form\\Form->handleRequest(Object(Symfony\\Component\\HttpFoundation\\Request))
#11 /var/www/html/app/bundles/EmailBundle/Controller/EmailController.php(701): Mautic\\CoreBundle\\Controller\\AbstractFormController->isFormValid(Object(Symfony\\Component\\Form\\Form))
#12 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(163): Mautic\\EmailBundle\\Controller\\EmailController->editAction(Object(Symfony\\Component\\HttpFoundation\\Request), Object(Mautic\\CoreBundle\\Twig\\Helper\\AssetsHelper), Object(Mautic\\CoreBundle\\Translation\\Translator), Object(Symfony\\Bundle\\FrameworkBundle\\Routing\\Router), Object(Mautic\\CoreBundle\\Helper\\CoreParametersHelper), '1', false, false)
#13 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(75): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw(Object(Symfony\\Component\\HttpFoundation\\Request), 2)
#14 /var/www/html/vendor/symfony/framework-bundle/Controller/AbstractController.php(156): Symfony\\Component\\HttpKernel\\HttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 2)
#15 /var/www/html/app/bundles/CoreBundle/Controller/CommonController.php(415): Symfony\\Bundle\\FrameworkBundle\\Controller\\AbstractController->forward('Mautic\\\\EmailBun...', Array, Array)
#16 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(163): Mautic\\CoreBundle\\Controller\\CommonController->executeAction(Object(Symfony\\Component\\HttpFoundation\\Request), 'edit', '1', 0, '')
#17 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(75): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw(Object(Symfony\\Component\\HttpFoundation\\Request), 1)
#18 /var/www/html/vendor/symfony/http-kernel/Kernel.php(202): Symfony\\Component\\HttpKernel\\HttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
#19 /var/www/html/app/AppKernel.php(109): Symfony\\Component\\HttpKernel\\Kernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
#20 /var/www/html/app/middlewares/CORSMiddleware.php(76): AppKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
#21 /var/www/html/app/middlewares/HSTSMiddleware.php(39): Mautic\\Middleware\\CORSMiddleware->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
#22 /var/www/html/app/middlewares/CatchExceptionMiddleware.php(28): Mautic\\Middleware\\HSTSMiddleware->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
#23 /var/www/html/app/middlewares/Dev/IpRestrictMiddleware.php(52): Mautic\\Middleware\\CatchExceptionMiddleware->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
#24 /var/www/html/app/middlewares/VersionCheckMiddleware.php(58): Mautic\\Middleware\\Dev\\IpRestrictMiddleware->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
#25 /var/www/html/app/middlewares/TrustMiddleware.php(42): Mautic\\Middleware\\VersionCheckMiddleware->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
#26 /var/www/html/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Mautic\\Middleware\\TrustMiddleware->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
#27 /var/www/html/index.php(19): Stack\\StackedHttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request))
#28 {main}
"} {"hostname":"mautic-web","pid":4125}
`

- Additionally, there are some modules that lack an icon. If necessary, I can create an icon following the same graphic style as the current ones.
Frame 50

- One observation is the inconsistency of icons between the LP builder and Email builder. The builders should maintain consistency in terms of icon usage. I suggest using the LP builder icons for both builders.
Frame 51

Overall, there is still some work needed on the email builder. I hope this feedback is helpful! You rockkk!

**Comment by LordRembo** [comment]
@code5rick I believe you'll have to reset or reinstall the Mautic you are testing on. The 500 error, as well as the template being blank (or some things not saving), is due to some bigger changes that were merged in 5.x. I had a bunch of issues after a rebase last week, which broke my local site with errors like that. But after reinstallation, it all works fine.

**Comment by code5rick** [comment]
@LordRembo I used GitPod for testing your PR

**Comment by LordRembo** [comment]
Hm, I'm seeing the 500 error as well in Gitpod but not locally. Okay, something to investigate further on Friday then.
EDIT:
I think the difference has to do with the PHP version. Note to self: 
* compare versions & look into compatibility issues with `iconv`or encoding in PHP8.2
* Also look for illegal character(s) in the Brienz theme

**Comment by LordRembo** [comment]
Hi @code5rick , I believe this latest push should address the bugs you reported. Maybe you could have another look?

What changed:
- missing icons are now set ([required an update](https://github.com/mautic/grapesjs-preset-mautic/pull/49) of the `grapesjs-mautic-preset` depencency)
- the icons in the top bar are now the same for both landing pages and emails (was also a part of that depencency)
- The difference for the Map width between Editor preview and Saved version, is also present in previous versions of Mautic. It's something to do with iFrame rendering, so I won't be fixing that in this PR. Feel free to make a new bug ticket for it though. I can try to pick that up in a separate PR.
- The errors with the Brienz theme + a selected theme returning an empty page, seem to have disappeared. They were not directly related to the GrapesJS changes, but encoding handled by PHP. That explains why it was not happening in my local install (php 8.0) but did happen in GitPod (which ran in php 8.2). Haven't been able to replicate it anymore.

**Comment by code5rick** [comment]
> * the icons in the top bar are now the same for both landing pages and emails (was also a part of that depencency)

✅ Correct, I tested it and it seems that the icons are the same on Email and LP themes.

> * The difference for the Map width between Editor preview and Saved version, is also present in previous versions of Mautic. It's something to do with iFrame rendering, so I won't be fixing that in this PR. Feel free to make a new bug ticket for it though. I can try to pick that up in a separate PR.

✅ Done, reported the issue: https://github.com/mautic/mautic/issues/13574

> * The errors with the Brienz theme + a selected theme returning an empty page, seem to have disappeared. They were not directly related to the GrapesJS changes, but encoding handled by PHP. That explains why it was not happening in my local install (php 8.0) but did happen in GitPod (which ran in php 8.2). Haven't been able to replicate it anymore.

⚠️ That bug seems still be present with the Brienz theme, displaying the same error as I reported previously. Steps to reproduce that bug:

1 - Navigate to Channels > Emails
2 - Create a new email
3 - Fill the subject
4 - Select the theme "Brienz" and next go to "Builder"
5 - Open the builder, and do not do anything inside of the builder. Close the builder.
6 - Click on "Save and Close". The loading top bar will stop and right after it will display this screen:
image

**Comment by patrykgruszka** [comment]
I've run into this bug with `iconv()` when testing other PR based on 5.x, so it may not be related to the changes you've made here.

**Comment by LordRembo** [comment]
Very strange. I'll give the brienz template a look next Friday, maybe there is an illegal character or some cleanup I can do.
But if the bug is indeed in 5.x as well, I'll make a separate issue for it and we can exclude it here from the test criteria.

**Review by patrykgruszka — COMMENTED** [review]
There's WIP PR which should address this issue: https://github.com/mautic/mautic/pull/13572

I've tested this updated version and found only a few problems with dynamic content, as described in a comment in the related PR: https://github.com/mautic/grapesjs-preset-mautic/pull/49#issuecomment-2022349184
Other features of the builder seem to work fine.

I've been struggling to rebuild the JS/CSS assets as the required nodejs version is different here. The ddev instance is operating on v10, while the dependencies require at least v16 for installation and asset rebuilding. Ideally, we should be able to handle asset rebuilding within the ddev shell, or alternatively, include detailed requirements and setup instructions in the GrapesJS readme file. What are your thoughts on this?

**Comment by LordRembo** [comment]
@patrykgruszka I somehow skipped over that the default in ddev was that old, sorry. I've now added an `.npmrc` file + engine check in package.json so users are now aware of the minimal requirements when they install the modules (it triggers a warning message).
NVM comes preinstalled in DDEV (`ddev nvm`) so it's easy enough for users to update it themselves when needed.

Once we have the GrapesJS updates merged into M5, we can see about updating the default in DDEV instead (and removing this `.npmrc`). I'd rather not do it in this PR because the CKE stuff also depends on Node and that should be dealt with separately from this PR.

**Comment by LordRembo** [comment]
Okay, so to recap:
- the encoding/status issues are related to encoding and present in 5.x as well (with an issue [and PR](https://github.com/mautic/mautic/pull/13572) to address it)
- there is 1 issue remaining with the GrapesJS update, and it is related to [the preset dependency](https://github.com/mautic/grapesjs-preset-mautic/pull/49): Dynamic Content Block output isn't quite right in the builder

I'll investigate that last issue in that repo, see what I come up with.

**Comment by LordRembo** [comment]
@patrykgruszka The issue with the Dynamic Content block should be fixed now, could you have a look if it's okay?
In preview, the html is slightly different than before but functionally and visually it is the same: a block where you can add/edit text/tokens.
I updated [the preset PR](https://github.com/mautic/grapesjs-preset-mautic/pull/49) and updated this PR to use the changes.  (I ignored the bugs that already exist in 5.X branch)

**Comment by LordRembo** [comment]
@code5rick Would you reviewing again? Keeping in mind that the bug with Brienz is related to [this ticket](https://github.com/mautic/mautic/issues/13571) and [this PR](https://github.com/mautic/mautic/pull/13572), the icons themselves should be fixed and more consistent now.

**Comment by code5rick** [comment]
@LordRembo tested your PR:
✅ Consistency with the icons, both on LP and Email.
✅ Missing icons in the modules have been fixed.
✅ Creation of a LP with the builder worked fine.
⚠️ As you mentioned, there are still some bugs present and already reported in other PRs, which could have an impact. One thing I noticed is that while editing the Paprika theme, the builder doesn't display the full elements, hiding some text. I will attach a screenshot:

image

This may not be directly related to this PR, as @patrykgruszka has made some corrections in another PR regarding a visibility bug with some GrapeJS modules.

Other than that, all good! 👍🏼

**Comment by LordRembo** [comment]
@code5rick Could you link the PR you mentioned? I'm not sure what's up with that visibility

**Comment by LordRembo** [comment]
@code5rick Found it. It's indeed nothing to do with this update. There's a syntax error in the Paprika template, a stray `;` that should be inside the quotes of the inline styling property. if @patrykgruszka indeed fixed it in another PR, I'll leave it as-is, otherwise I'll make a separate issue and PR for it.

**Edit**: [a new issue added](https://github.com/mautic/mautic/issues/13618), as well as [a PR to fix it](https://github.com/mautic/mautic/pull/13619).

**Edit 2**: it's been merged

**Review by patrykgruszka — APPROVED** [review]
Works as expected after recent changes 👍

**Comment by LordRembo** [comment]
The branch conflict is due to [the Apply button fix getting merged into 5.x](https://github.com/mautic/mautic/pull/13665), because it makes changes to some of the same files (most of them build files).  
This just means that the files in GrapesJSBuilderBundle needs to be rebuilt after merging. Or better: give priority to the changes of this upgrade PR, because they contain the same effectual changes.
It also implies that [the depending preset](https://github.com/mautic/grapesjs-preset-mautic/pull/49) has to be merged so I can replace the reference in the package file. But we knew that already.

**Comment by LordRembo** [comment]
@code5rick If the issues you mentioned before are fixed to your liking, could you give the PR a passing review? Then we have the required amount of reviews to get this ready to commit.

**Review by code5rick — APPROVED** [review]

**Comment by LordRembo** [comment]
Cool! We're 2 for 2 on the reviews. I'd just like to wait for [the preset dependency](https://github.com/mautic/grapesjs-preset-mautic/pull/49) to be merged, so I can switch the module url. And then I think this can be marked as ready-to-commit.

**Comment by escopecz** [comment]
I merged the preset PR. But there are some conflicts in this one.

**Comment by LordRembo** [comment]
@escopecz I replaced the preset branch and switched the base branch to 5.1, seeing as that's the milestone release this is intended for. That got rid of the conflicts for now.
Since compiled JS is part of the GrapesjsBuilderBundle, that's gonna give conflicts on every merge with branches that also contain JS changes in the same bundle. Not much we can do about that, other than resolve the conflict after merging (normally just reinstalling the node modules and rebuilding).

**Comment by LordRembo** [comment]
@escopecz Okay, switching to 5.1 as base branch just gave me a lot of unwanted changes, so I put it back on 5.x. Not sure how to proceed.

**Comment by escopecz** [comment]
Right. We used 5.1 before 5.0 was released so we could merge PRs without waiting for the 5.0 release. Now the 5.1 changes live in 5.x branch as per our normal branching strategy. The 5.1 branch will get reset to 5.x once we release 5.1. Sorry for the confusion.

**Comment by LordRembo** [comment]
@escopecz Ok, I see, so then I'll leave the branch on `5.x`. 
I don't know why this is still showing conflicts here, it doesn't when I compare locally. I just tried merging locally and everything looks fine. My changes for those files are the 'newest', so they should take precedence.

**Comment by LordRembo** [comment]
Merge issues are gone now, it seems. I just fetched everything again from scratch, merged, rebuild and pushed. No actual changes done to what what I committed before. Git is weird.

**Review by RCheesley — APPROVED** [review]
Tested both email and landing page builder and all working as expected! Thanks for persevering with this @LordRembo ! 🚀

**Pull request merged**